art/attacks/inference/membership_inference/black_box.py

```diff
@@ -74,7 +74,7 @@ def __init__(
         :param attack_model: The attack model to train, optional. If none is provided, a default model will be created.
         """
 
-        super().__init__(estimator=classifier)
+        super().__init__(classifier=classifier)
```

Comment:
Could this base class ever be used for attacks on non-classification models?
```suggestion
        super().__init__(estimator=classifier)
```